src/main/java/com/modrinth/minotaur/TaskModrinthUpload.java

```diff
@@ -221,4 +188,72 @@ public void apply() {
 			}
 		}
 	}
+
+	private void autoDetectLoaders(ModrinthExtension ext, PluginManager pluginManager) {
+		Map<String, String> pluginLoaderMap = new HashMap<>();
+		pluginLoaderMap.put("net.minecraftforge.gradle", "forge");
+		pluginLoaderMap.put("org.quiltmc.loom", "quilt");
+		pluginLoaderMap.put("org.spongepowered.gradle.plugin", "sponge");
+		pluginLoaderMap.put("io.papermc.paperweight.userdev", "paper");
+		pluginLoaderMap.put("xyz.jpenilla.run-paper", "paper");
+		pluginLoaderMap.put("xyz.jpenilla.run-waterfall", "waterfall");
+		pluginLoaderMap.put("xyz.jpenilla.run-velocity", "velocity");
+
+		pluginLoaderMap.forEach((plugin, loader) -> {
+			if (pluginManager.hasPlugin(plugin)) {
+				getLogger().debug("Adding loader '{}' because plugin '{}' was found.", loader, plugin);
+				add(ext.getLoaders(), loader);
+			}
+		});
+
+		if (!ext.getLoaders().get().contains("quilt") && getProject().getExtensions().findByName("loom") != null) {
+			String loomPlatform = getProject().getProviders().gradleProperty("loom.platform").getOrNull();
+			if (loomPlatform != null && loomPlatform.equalsIgnoreCase("forge")) {
+				getLogger().debug("Adding loader 'forge' because 'loom' extension was found and loom.platform=forge.");
```

Comment:
Would be a lot easier to just use the value of loom.platform, because it's also possible for this to be Quilt using Architectury.

Reply:
Didn't know loom.platform could be `quilt`, will update this